controllers/hetznercluster_controller.go

```diff
@@ -203,7 +203,9 @@ func (r *HetznerClusterReconciler) reconcileNormal(ctx context.Context, clusterS
 		return reconcile.Result{}, fmt.Errorf("failed to reconcile placement groups for HetznerCluster %s/%s: %w", hetznerCluster.Namespace, hetznerCluster.Name, err)
 	}
 
-	if hetznerCluster.Spec.ControlPlaneLoadBalancer.Enabled {
+	if !hetznerCluster.Spec.ControlPlaneLoadBalancer.Enabled && (hetznerCluster.Spec.ControlPlaneEndpoint == nil || hetznerCluster.Spec.ControlPlaneEndpoint.Host == "") {
+		clusterScope.Logger.Info("disabled LoadBalancer and not yet provided ControlPlane endpoint")
```

Comment:
can you add a new condition and set it on false here? 
Maybe something like "ControlPlaneEndpointSet". You can then set it on true or false in these cases below. 
If the cluster is usable, it should be always on true AFAIK.